mayastor/src/lvs/lvs_pool.rs

```diff
@@ -243,25 +257,130 @@ impl Lvs {
         }
     }
 
+    /// imports a pool based on its name, uuid and base bdev name
+    #[tracing::instrument(level = "debug", err)]
+    pub async fn import_from_args(args: PoolArgs) -> Result<Lvs, Error> {
+        if args.disks.len() != 1 {
+            return Err(Error::Invalid {
+                source: Errno::EINVAL,
+                msg: format!(
+                    "invalid number {} of devices {:?}",
+                    args.disks.len(),
+                    args.disks
+                ),
+            });
+        }
+
+        // default to aio if no specific driver scheme is specified
+        let disks = args
+            .disks
+            .iter()
+            .map(|d| {
+                if Url::parse(d).is_err() {
+                    format!("aio://{}", d)
+                } else {
+                    d.clone()
+                }
+            })
+            .collect::<Vec<_>>();
```

Comment:
I think we may be able to simplify this code a bit, since we already know that we expect 1 and only 1 disk. So rather than doing the check with `if args.disks.len() != 1 {` we might be able to:
```rust
  let disk = match args.disks.first() {
      Some(disk) if args.disks.len() == 1 => {
          disk
      },
      _ => return err "LEN NOT 1",
  };
```